### PR TITLE
PROXY-11

### PR DESCRIPTION
**Pull request by denStepaniuk**
Added POST endpoint to retrive img from distant server

### CODE REVIEW

File: delivery/router/router.js [hunk 1]
```diff
@@ -1,24 +1,42 @@
...
 
 const router = express.Router();
+router.use(express.json());
 
 router.get('/meteors', (req, res) => {
   if (Object.keys(req.query).length !== 0) {
     retrieveMeteorDataWithQueryParams(res, req.query)
     .catch((err) => {
-      processError(err,res);
+      const processedError = processError(err,res);
```
Comment (mdziarkach) [inline delivery/router/router.js]: Please check our chat
I already described how to handle errors

File: app-requests.http [hunk 2]
```diff
@@ -1,27 +1,38 @@
...
+
+###POST obtain photo
+POST http://localhost:4000/yep-its-post-but-get-me-photo
+Content-Type: application/json
+
+{
+"id": 1234,
+"name": "Especto Patronum!",
+"user_api_key": "",
+"earth_date": "2015-6-3"
```
Comment (plutsenka) [inline app-requests.http]: As described in task, we should return most recent photo, so you can remove `earth_date` parameter and resolve recent photo on server side

File: use-cases/retrieve-meteors-data.js [hunk 3]
```diff
@@ -56,7 +57,20 @@ const retrieveMeteorDataWithQueryParams = async (serverResp, queryParams) => {
...
+  .then((response) => {
+    return retrieveLinkOfPictureOfTheDay(response.data);
+  })
+};
+
 module.exports = {
   retrieveMeteorDataLastWeek,
-  retrieveMeteorDataWithQueryParams
+  retrieveMeteorDataWithQueryParams,
+  getLinkOnRoverPicture
```
Comment (plutsenka) [inline use-cases/retrieve-meteors-data.js]: Link to, probably

Comment (denStepaniuk) [reply use-cases/retrieve-meteors-data.js]: Thanx! Fixed.

File: app-requests.http (hunk 2 above)
Comment (denStepaniuk) [reply app-requests.http]: Thanx, redone.

File: delivery/router/router.js (hunk 1 above)
Comment (denStepaniuk) [reply delivery/router/router.js]: Thanks, redone .

**Pull request merged**